charts/simpledataexchanger/values.yaml

```diff
@@ -26,9 +26,9 @@ sdepostgresql:
   enabled: true
   fullnameOverride: "product-sde-postgres"
   auth:
-    postgresPassword: "default"
+    postgresPassword: ""
```

Comment:
Is there a random password generated here? Can you explain how does it work?

Reply:
This change was [initiated by me](https://github.com/eclipse-tractusx/managed-simple-data-exchanger/issues/22). If the postgresPassword or password are left empty the bitnami chart [will auto-generate](https://github.com/bitnami/charts/blob/6ec633ba194236430b35957024e242b4373c5e58/bitnami/postgresql/values.schema.json#L25) a  random password

Reply:
> Is there a random password generated here? Can you explain how does it work?

-  Yes Harald is right, bitnami charts will trigger the random password .If we don't provide the password it will create   secret with random password and we have handle this in  backend deployment file.